content/en/docs/instrumentation/net/exporters.md

```diff
@@ -214,28 +224,35 @@ services:
 ```csharp
 var builder = WebApplication.CreateBuilder(args);
 
-builder.Services.AddOpenTelemetry().WithMetrics(b =>
-{
-    b
-    .AddPrometheusExporter(options =>
-    {
-        options.ScrapeResponseCacheDurationMilliseconds = 0;
-    })
-    // The rest of your setup code goes here too
-});
+builder.Services.AddOpenTelemetry()
+  .WithMetrics(b => b.AddPrometheusExporter());
 ```
 
-Otherwise, configure the exporter when creating a meter provider:
+You'll then need to add the endpoint so that Prometheus can scrape your site.
+You can do this using the `IAppBuilder` extension like this:
 
 ```csharp
-using var tracerProvider = Sdk.CreateMeterProviderBuilder()
-    .AddPrometheusExporter(options =>
-    {
-        options.ScrapeResponseCacheDurationMilliseconds = 0;
-    })
+var builder = WebApplication.CreateBuilder(args);
 
-    // Other setup code, like setting a meter goes here
+// .. Setup
 
+var app = builder.Build();
+
+app.UseOpenTelemetryPrometheusScrapingEndpoint();
+
+await app.RunAsync();
+```
+
+### Non-ASP.NET Core
+
+For applications not using ASP.NET Core, you can use the `HttpListner` version.
```

Comment:
need to show the package install part for the HttpListner

Reply:
Added